app/controllers/comments_controller.rb

```diff
@@ -1,26 +1,28 @@
 class CommentsController < ApplicationController
   def index
-    comments = Comment.all.order({ :created_at => :asc })
+    matching_comments = Comment.all.order({ :created_at => :asc })
 
-    render({ :json => comments.as_json })
+    render({ :json => matching_comments.as_json })
```

Comment:
Woah `json`? I don't think we've ever asked them to do this during summer quarter. I don't believe the current target does this either.

Reply:
Yeahhh this was my bad.  Should be good now.